src/bin/index.ts

```diff
@@ -12,7 +12,7 @@ import { crypto, pkijs, asn1js } from "../crypto";
 
 (global as any)["DOMParser"] = DOMParser;
 (global as any)["XMLSerializer"] = XMLSerializer;
-XAdES.Application.setEngine("@peculiar/webcrypto", crypto);
+XAdES.Application.setEngine("@peculiar/webcrypto", crypto as Crypto);
```

Comment:
Required since it seems Webcrypto's Crypto interface is slightly different from native Crypto in some stage detail:
```
src/formats/eutl.ts:8:52 - error TS2345: Argument of type 'import("/Users/worldthirteen/Documents/Projects/pv/tl-create/node_modules/@peculiar/webcrypto/index").Crypto' is not assignable to parameter of type 'Crypto'.
  Types of property 'getRandomValues' are incompatible.
    Type '<T extends Uint8Array | Int8Array | Int16Array | Int32Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | null>(array: T) => T' is not assignable to type '<T extends ArrayBufferView | null>(array: T) => T'.
      Types of parameters 'array' and 'array' are incompatible.
        Type 'T' is not assignable to type 'Uint8Array | Int8Array | Int16Array | Int32Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | null'.
          Type 'ArrayBufferView | null' is not assignable to type 'Uint8Array | Int8Array | Int16Array | Int32Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | null'.
            Type 'ArrayBufferView' is not assignable to type 'Uint8Array | Int8Array | Int16Array | Int32Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | null'.
              Type 'ArrayBufferView' is missing the following properties from type 'DataView': getFloat32, getFloat64, getInt8, getInt16, and 17 more.
                Type 'T' is not assignable to type 'DataView'.
                  Type 'ArrayBufferView | null' is not assignable to type 'DataView'.
                    Type 'null' is not assignable to type 'DataView'.

8 XAdES.Application.setEngine("@peculiar/webcrypto", crypto);
```
so I was forced to add this `as`.

Reply:
The problem is that TS changes WebCrypto interfaces from version to version. We need to upgrade TS package version for all of our crypto modules